algorithmic_efficiency/checkpoint_utils.py

```diff
@@ -160,7 +160,8 @@ def save_checkpoint(framework: str,
     else:
       save_path = os.path.join(checkpoint_dir, 'checkpoint')
       torch.save(checkpoint_state, save_path)
-    logging.info('Saved checkpoint to %s', save_path)
 
   if USE_PYTORCH_DDP:
     dist.barrier()
+  if RANK == 0:
```

Comment:
`logging.info()` will not do anything in any RANK != 0 anyway, see [here](https://github.com/mlcommons/algorithmic-efficiency/blob/main/algorithmic_efficiency/pytorch_utils.py#L41-L46), so this change is not necessary I think.

Reply:
I originally didn't have the check, but then `save_path` isn't defined so I had to add it.

Reply:
Ah right, I was too tired to catch that.